RetroFE/Source/Execute/Launcher.cpp

```diff
@@ -52,14 +52,14 @@ bool Launcher::run(std::string collection, Item *collectionItem, Page *currentPa
         std::string line;
         if (std::getline(launcherStream, line)) {
             // Construct localLauncher key
-            std::string localLauncherKey = "localLaunchers." + collection + "." + Utils::toLower(line);
```

Comment:
I'm nervous about not lower casing the launcher names for consistency

Reply:
But I see you removed from other places so I guess people will need to know it's case sensitive